internal/fuzzbinder/util.go

```diff
@@ -0,0 +1,45 @@
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package fuzzbinder
+
+// This file provides utility functions for FuzzBinder tool.
+
+import (
+	"fmt"
+	"time"
+)
+
+// ValidateFuzzingDate validates that the fuzzing date chosen to generate the fuzzing
+// claims is no more than 15 days prior to the date of execution of FuzzBinder cmd
+// and not in the future.
+func ValidateFuzzingDate(date string, currentTime time.Time) error {
```

Comment:
```suggestion
func ValidateFuzzingDate(date string, referenceTime time.Time) error {
```

I am not sure `currentTime` is an accurate name.

Reply:
It is the time when the FuzzBinder cmd is executed. I changed it to `referenceTime` if it's more clear.